packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue

```diff
@@ -136,6 +136,15 @@ export default defineComponent({
         return false
       }
 
+      const pattern =
+        "^[a-zA-Z_][a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]*(@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*)*$"
```

Comment:
Since this is correct but hard to read, could you please add a small comment explaining that this regex comes from the server?

Reply:
Done